container/help.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 
-DOCKER="/usr/bin/docker"
 SELF=$1
```

Comment:
I would remove the SELF variable, because its contents are provided by the user and then printed back to the user few lines below, so there is no real added value.

Reply:
OK, I agree with the change.